public/components/marketing-pricings/1.html

```diff
@@ -0,0 +1,116 @@
+                <strong class="text-3xl sm:text-4xl">20$</strong>
+                <span> / month</span>
+            </div>
+        </div>
+        <ul class="flex sm:text-base text-sm flex-col gap-2">
+            <li class="flex gap-1 items-center">
+                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="#4F46FF" class="w-5 h-5">
+                    <path stroke-linecap="round" stroke-linejoin="round" d="M4.5 12.75l6 6 9-13.5" />
+                </svg>
+                <span>
+                    10 users included
+                </span>
+            </li>
+            <li class="flex gap-1 items-center">
+                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="#4F46FF" class="w-5 h-5">
+                    <path stroke-linecap="round" stroke-linejoin="round" d="M4.5 12.75l6 6 9-13.5" />
+                </svg>
+                <span>
+                    2 GB of storage
+                </span>
+            </li>
+            <li class="flex gap-1 items-center">
+                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="#4F46FF" class="w-5 h-5">
+                    <path stroke-linecap="round" stroke-linejoin="round" d="M4.5 12.75l6 6 9-13.5" />
+                </svg>
+                <span>
+                    Email support
+                </span>
+            </li>
+            <li class="flex gap-1 items-center">
+                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="#4F46FF" class="w-5 h-5">
+                    <path stroke-linecap="round" stroke-linejoin="round" d="M4.5 12.75l6 6 9-13.5" />
+                </svg>
+                <span>
+                    Help center access
+                </span>
+            </li>
+        </ul>
+        <a
+                class="inline-block rounded border border-indigo-600 bg-indigo-600 px-12 py-3 text-sm font-medium text-white hover:bg-transparent hover:text-indigo-600 focus:outline-none focus:ring active:text-indigo-500"
+                href="/download"
+        >
+            Get Started
+        </a>
+    </div>
+    <div class="bg-indigo-600 text-white shadow-lg border border-gray-200 flex flex-col gap-6 sm:p-10 lg:p-12 p-8 rounded-2xl">
+        <div class="flex flex-col sm:gap-3 gap-1">
+            <p class="text-2xl sm:text-3xl">PRO</p>
+            <div class="flex items-center gap-2 justify-center">
+                <strong class="text-3xl sm:text-4xl">30$</strong>
+                <span> / month</span>
+            </div>
+        </div>
+        <ul class="flex flex-col gap-2 sm:text-base text-sm">
+            <li class="flex gap-1 items-center">
+                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="#ffffff" class="w-5 h-5">
```

Comment:
Thanks for changing to Heroicons, but `stroke` should be using `currentColor` not a hardcoded value.